### PR TITLE
[C-2102] Improve online/offline transitions

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Improves component animations/transitions when going from online to offline and vis versa. happy to post a video if thatll make it more clear

### CODE REVIEW

File: packages/mobile/src/screens/favorites-screen/OfflineContentBanner.tsx [hunk 1]
```diff
@@ -43,7 +44,7 @@ export const OfflineContentBanner = () => {
   const isReachable = useSelector(getIsReachable)
   if (!isOfflineModeEnabled || isReachable) return null
   return (
-    <View>
+    <Animated.View exiting={SlideOutUp.duration(1000)}>
```
Comment (amendelsohn) [inline packages/mobile/src/screens/favorites-screen/OfflineContentBanner.tsx:47]: should it slide in too?

**Review by amendelsohn — APPROVED** [review]
Oh that's super easy. Very cool!

File: packages/mobile/src/screens/favorites-screen/OfflineContentBanner.tsx (hunk 1 above)
Comment (dylanjeffers) [reply packages/mobile/src/screens/favorites-screen/OfflineContentBanner.tsx:47]: So when experimenting it slides in due to the surrounding layout transition, but needed this for the disappearance to look smooth. lets def play with it on device when up to make sure it looks clean

Comment (amendelsohn) [reply packages/mobile/src/screens/favorites-screen/OfflineContentBanner.tsx:47]: ah, okay cool

**Pull request merged**